papis/utils.py

```diff
@@ -71,9 +71,22 @@ def general_open(file_name: str,
         subprocess.call(cmd)
     else:
         logger.debug("Not waiting for process to finish")
-        subprocess.Popen(
-            cmd, shell=False,
-            stdin=None, stdout=None, stderr=None, close_fds=True)
+        popen_kwargs = {
+            "shell": False,
+            "stdin": None,
+            "stdout": None,
+            "stderr": None,
+            "close_fds": True
+        }
+        # Tell subprocess to detach the process.
+        if sys.platform == "win32":
+            popen_kwargs["creationflags"] = subprocess.DETACHED_PROCESS
+            popen_kwargs["creationflags"] |= subprocess.CREATE_NEW_PROCESS_GROUP
```

Comment:
Why are these Windows only? Does it not apply to Unix shells?

Reply:
Ah, just noticed that these are Windows-only constants. Nevermind!

Reply:
Exactly, no worries :-)